axelor-account/src/main/resources/views/PaymentVoucher.xml

```diff
@@ -189,7 +189,7 @@
     <action name="action-attrs-payment-voucher-readonly-status"/>
     <action name="action-payment-voucher-method-fill-bank-details"/>
     <action name="action-payment-voucher-record-value-for-collection-account"
-      if="paymentMode.typeSelect == 7"/>
+      if="paymentMode &amp;&amp; paymentMode.typeSelect == 7"/>
```

Comment:
This a groovy expression, you can just use a "?", like "paymentMode?.typeSelect == 7"